Header header
string robot_id
float64[] ctrl_pts
float64 pvar
uint8 spl_deg
float64 time_to_exec
float64 radius
